liquidctl/driver/kraken3.py

```diff
@@ -818,26 +872,35 @@ def _send_data(self, data, bulkInfo):
         self._write_then_read([0x36, 0x03])  # unknown
 
         buckets = self._query_buckets()  # query all buckets and store their response
-
-        self._write_then_read([0x20, 0x03])  # unknown
-        self._write_then_read([0x74, 0x01])  # keepalive
-        self._write_then_read([0x70, 0x01])  # unknown
-        self._write_then_read([0x74, 0x01])  # keepalive
-
```

Comment:
Have you confirmed that these are indeed no longer necessary even for the older Kraken Zs?

Reply:
Oh yes they can all be removed. Only 0x36,0x3 is useful (sort of close any open incomplete transfer)